panther_analysis_tool/rule.py

```diff
@@ -93,164 +93,6 @@
 )
 
 
-# pylint: disable=too-many-instance-attributes,unsubscriptable-object
```

Comment:
Removing the classes from here will break development AWS deployments, since they use latest master by default (maybe we should change this). You can either freeze the commit in panther-enterprise (see `panther_config.yml`), or make the necessary change there too first and remove these in a subsequent PR.